subxt/src/rpc.rs

```diff
@@ -332,7 +243,7 @@ impl<T: Config> Rpc<T> {
         key: &StorageKey,
         hash: Option<T::Hash>,
     ) -> Result<Option<StorageData>, BasicError> {
-        let params = &[to_json_value(key)?, to_json_value(hash)?];
+        let params = rpc_params![key, hash];
```

Comment:
NOTE; this panics if the serialization fails

Reply:
I guess there should be no reason for these things to fail to serialize (afaik it's just maps where you have to be careful with json serializing?)

Reply:
indeed,

I'm working on a follow-up to replace this with proc macros to get rid of this anyway, so view this as temporary.

Reply:
Yes these are infallible